src/hue_slider.cpp

```diff
@@ -129,11 +126,17 @@ void HueSlider::setFullColor(const QColor& color)
 
 qreal HueSlider::colorHue() const
 {
-    return p->percent();
+    auto hue = qreal(value() - minimum()) / (maximum() - minimum());
```

Comment:
This would result into a division by zero if maximum() == minimum(). I think it's worth having a check here.

Reply:
Good catch.

Reply:
On the other hand it doesn't make sense to set max == min. I wonder what hue should slider return then? Previous version has the same /0 behaviour.

I guess we can just return 0 (red) since that would at least match both min and max.

Reply:
Perhaps the best would be to forbid manually changing min/max at all, but since setters aren't virtual it's not going to work with abstract slider pointers.

Reply:
I think the easiest way is to have a condition here, returning 0 seems reasonable.